### PR TITLE
feat(Table): changed Date filtering to allow multiple or single selection

### PR DESCRIPTION
**Pull request by MagicAardvark**
allows date filters to be toggled between multiple and single selection. with multiple=true property on meta data.

##### **What did you change?**
Table.js and spec file.

Changed includes to indexOf check due to PhantomJS not supporting includes, no shim in polyfills

##### **Reviewers**
* @bvkimball @jgodi @krsween 

##### **Checklist (completed via merger)**
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [contributing guide](https://github.com/bullhorn/novo-elements/blob/master/CONTRIBUTING.md)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Visually tested in supported browsers and devices

### CODE REVIEW

File: src/elements/table/Table.js [hunk 1]
```diff
@@ -259,8 +259,8 @@ export class NovoTable {
      * @param filter
      */
     onFilterClick(column, filter) {
-        if (Array.isArray(column.filter)) {
-            if (column.filter.includes(filter)) {
+        if (Array.isArray(column.filter) && column.multiple) {
```
Comment (jgodi) [inline src/elements/table/Table.js]: Does this break the other table filters that use select boxes and were multiple by default?

File: src/elements/table/Table.js [hunk 2]
```diff
@@ -259,8 +259,8 @@ export class NovoTable {
...
+            if (column.filter.indexOf(filter) > -1) {
```
Comment (krsween) [inline src/elements/table/Table.js]: Can you tilde this instead? I think it's a preference:

`if (column.filter.indexOf(filter) > -1) {`

`if (~column.filter.indexOf(filter)) {`

[Fiddle](https://jsfiddle.net/krsween/kf7q5cjL/)

File: src/elements/table/Table.js (hunk 1 above)
Comment (krsween) [inline src/elements/table/Table.js]: Not to pile on, but whatever the decision for multiple by default or single, can we call that out in the readme? 

https://github.com/bullhorn/novo-elements/blob/master/src/elements/table/README.md

Comment (jgodi) [inline src/elements/table/Table.js]: Just tested it and it does change the current functionality for selection based filters. Check the Table -> Basic -> Filter Status Column. This was multiple by default.

I am fine with making it single by default and making the `multiple=true`, but you would have to go through all the columns that are currently inside all Novo apps and make sure they get `multiple=true` if they were before.

@krsween @bvkimball thoughts?

**Pull request merged**